sdk/search/search-documents/src/indexDocumentsBatch.ts

```diff
@@ -69,14 +69,42 @@ export class IndexDocumentsBatch<T> {
    * @param keyName The name of their primary key in the index.
    * @param keyValues The primary key values of documents to delete.
    */
-  public delete(keyName: keyof T, keyValues: string[]): void {
-    const batch = keyValues.map<IndexDocumentsAction<T>>((keyValue) => {
-      return {
-        __actionType: "delete",
-        [keyName]: keyValue
-      } as IndexDocumentsAction<T>;
-    });
+  public delete(keyName: keyof T, keyValues: string[]): void;
 
-    this.actions.push(...batch);
+  /**
+   * Delete a set of documents by their primary key.
```

Comment:
nit: I think this can just say "Delete a set of documents."

Reply:
Done